### DISCUSSION TITLE
BaseActivity.Current should be nullable

### DISCUSSION THREAD

**Discussion by EricKwon94** [unoplatform/uno#18566] [Q&A] [answered]
The `BaseActivity.Current` is defined as a non-nullable property, but in certain scenarios, such as when the app is in the background, it returns null.

Why is the return value marked as non-nullable when it can potentially return null in these cases? Shouldn't this property be marked as nullable to prevent potential crashes?

**Comment by morning4coffe-dev** [comment, marked as answer]
Hey @EricKwon94,

Thank you for your suggestion, I think as well it should be nullable. I will make an Issue out of this and take a look.
